packages/blade/src/components/List/listTokens.ts

```diff
@@ -63,35 +59,35 @@ const listItemBulletMarginTop: Record<
     },
     onMobile: {
       small: 'spacing.2',
-      medium: 6,
+      medium: makeSize(6),
     },
   },
   ordered: {
     onDesktop: {
-      small: 0,
-      medium: 0,
+      small: makeSize(0),
+      medium: makeSize(0),
     },
     onMobile: {
-      small: 0,
-      medium: 0,
+      small: makeSize(0),
```

Comment:
maybe direct string value?

Reply:
Resolving this comment to avoid duplicates. Lets discuss it here - https://github.com/razorpay/blade/pull/1009#discussion_r1119660956